requirements-dev.txt

```diff
@@ -5,3 +5,4 @@ flake8
 mock
 pre-commit
 pytest
+pylint
```

Comment:
this file is only used for development dependencies -- runtime dependencies go in `setup.cfg`